Index: schema/mappings/document.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- schema/mappings/document.js	(revision c242e84c866289ff03204152935b5136c5c129e4)
+++ schema/mappings/document.js	(date 1591635069000)
@@ -146,7 +146,7 @@
 
     // geography
     center_point: require('./partial/centroid'),
-    shape: require('./partial/shape'),
+    polygon: require('./partial/shape'),
     bounding_box: require('./partial/boundingbox'),
 
     // meta info
@@ -192,7 +192,7 @@
     }
   }],
   _source: {
-    excludes : ['shape','phrase']
+    excludes : ['polygon','phrase']
   },
   dynamic: 'strict'
 };
